Formula/postgraphile.rb

```diff
@@ -6,7 +6,7 @@ class Postgraphile < Formula
   url "https://registry.npmjs.org/postgraphile/-/postgraphile-4.12.11.tgz"
   sha256 "553191171d304b35846d8fc8c40beace5649f85982d4363da13b992fd2aad3d3"
   license "MIT"
-  revision 1
+  revision 2
```

Comment:
We could probably drop the `revision`; this doesn't automatically uninstall `libpq` anyway.

Reply:
true